File: WEBAPI/Models/DTOs/Shipping/Econt/CountryDTO.cs
namespace Models.DTOs.Shipping.Econt
{
    public class CountryDTO
    {
        public int? Id { get; set; }
        public string? Code3 { get; set; }
        public string? Code2 { get; set; }
        public string? Name { get; set; }
        public string? NameEn { get; set; }
        public bool? IsEU { get; set; }
    }
}

File: PROJECT/FRONTEND/Common/DTOs/CountryDTO.ts


export class CountryDTO { 
    constructor(obj: Partial<CountryDTO>) {
        Object.assign(this,obj);
    }

    
    public id: number | null;
    public code3: string | null;
    public code2: string | null;
    public name: string | null;
    public nameEn: string | null;
    public isEU: boolean | null;
}
